src/utils/getNumberOfFileChanges.ts

```diff
@@ -0,0 +1,27 @@
+import * as vscode from "vscode";
+import getFullBlame from "./getFullBlame";
+
+async function getNumberOfFileChanges(gitAPI:any, lineNumber:number) {
```

Comment:
If this returns on the file, why do we need to send the _line_?

Reply:
How I imagine it is that once you put this on the content box, you will pass the hovered line index as the parameter value for lineNumber. 

How do you imagine it?

Reply:
Well, that would be another function, right?
If hovering over _any_ line returns the same value, why do I pass the line param?